Dockerfile-installer.arm32v7

```diff
@@ -0,0 +1,48 @@
+# Specify the version of PHP we use for our Chevereto
+ARG PHP_VERSION=7.3-apache
+FROM alpine as downloader
+
+RUN apk add --no-cache curl && \
+    mkdir -p /extracted/Chevereto/images && \
+    cd /extracted/Chevereto && \
+    curl -sS -o index.php -L "https://chevereto.com/download/file/installer" && \
+    curl -L https://github.com/balena-io/qemu/releases/download/v4.0.0%2Bbalena2/qemu-4.0.0.balena2-arm.tar.gz | tar zxvf - -C .  && \
```

Comment:
Why do we have to include all these binaries from qemu? Why can't we just use the one from underlying `php` image?